src/components/component-playground.jsx

```diff
@@ -554,48 +558,44 @@ module.exports = React.createClass({
     });
   },
 
-  _getComponentsWithMatchingFixtures() {
-    return _.pick(_.mapValues(this.props.components, function(details) {
-      return _.assign({}, details, {
-      // Only select fixtures matching search text and rebuild the object.
-        fixtures: _.pick(details.fixtures,
-            function(fixtureProps, fixtureName) {
-              return fixtureName.indexOf(this.state.searchText) !== -1;
-            }.bind(this)
-        )
-      });
-    }.bind(this)),
+  _getFilteredComponents() {
+    if (this.state.searchText.length < 2) {
+      return this.props.components;
+    }
+
+    // Get components which match search text by one or more fixture names, only
+    // keeping those fixtures. Also keep the selected fixture.
+    return _.pick(_.mapValues(this.props.components,
+        function(details, componentName) {
+          var maxScore = 0;
+          return _.assign({}, details, {
+          // Only select fixtures matching search text and rebuild the object.
+            fixtures: _.pick(details.fixtures,
+                function(fixtureProps, fixtureName) {
```

Comment:
Same as [this](https://github.com/uberVU/react-component-playground/pull/8/files#r74903210)